developers/development/guidelines.md

```diff
@@ -34,12 +37,15 @@ Note that this list also serves as a checklist for code reviews on pull requests
 1. The manifest must not contain any "Require-Bundle" entries. Instead, "Import-Package" must be used.
 1. The manifest must not export any internal package.
 1. The manifest must not have any version constraint on package imports, unless this is thoughtfully added. Note that Eclipse automatically adds these constraints based on the version in the target platform, which might be too high in many cases.
-1. The manifest must include all services in the Service-Component entry. A good approach is to put `OSGI-INF/*.xml` in there.
+1. The manifest must include all services in the Service-Component entry. A good approach is to put OSGI-INF/*.xml in there.
```

Comment:
@kaikreuzer the code fences change here confuses GitHub to think the rest of the text should be formatted in italic letters.

Reply:
Sorry, I actually didn't mean to remove them. Fixed now.